cfme/tests/containers/test_search_bar.py

```diff
@@ -73,4 +73,6 @@ def test_search_bar(provider, soft_assert):
                         'Should not find records, search results: "{}"'
                         .format(search_string, results_row_names))
         finally:
-            search.ensure_no_filter_applied()
+            # search.ensure_no_filter_applied() -> "TimedOutError: Could not do Waiting
+            # for advanced search to open at...", so better for this test to cleanup by:
+            search.normal_search("")
```

Comment:
The right way is to fix this issue but I really have to merge something work first, is this worth to open an (framework) issue for that? (if so, I'll open and change the comment to point on that):
 search.ensure_no_filter_applied() -> "TimedOutError: Could not do Waiting for advanced search to open at....
Anyway it's not applicable right now.